gpustack/server/controllers.py

```diff
@@ -551,15 +551,16 @@ async def _reconcile(self, event):
                         # container is still running, need not delete
                         continue
                     instance_names.append(instance.name)
-                    await ModelInstanceService(session).delete(instance)
+                    await instance.delete(session, auto_commit=False)
```

Comment:
Is this something we can improve in ModelInstanceService, so that cache update is involved?